app/controllers/practices_controller.rb

```diff
@@ -449,7 +451,9 @@ def practices_json(practices)
 end
 
 def create_date_initiated (date_initiated)
-  Date.new(date_initiated[:year].to_i, date_initiated[:month].to_i)
+  if date_initiated[:year].present? && date_initiated[:month].present?
```

Comment:
thanks for fixing this

Reply:
i would probably check if there is a `date_initiated` just in case
```suggestion
  if date_initiated && (date_initiated[:year].present? && date_initiated[:month].present?)
```

Reply:
Ok, sounds good!